audiomentations/augmentations/wind_absorption.py

```diff
@@ -0,0 +1,167 @@
+from audiomentations.core.transforms_interface import BaseWaveformTransform
+import numpy as np
+import librosa
+
+
+class WindAbsorption(BaseWaveformTransform):
+    """
+    Applies a wind absorption transform parametrized by temperature, humidity, and distance
+    of source.
+
+    This is not a scientifically accurate transform but basically applies a uniform
+    filterbank with attenuations given by:
+
+    att = exp(- distance * absorption_coefficient)
+
+    where distance is the microphone-source assumed distance in meters and `absorption_coefficient`
+    is adapted from a look up table by pyroomacoustics [1].
+
+    Note: This only "simulates" room absorption, and does not apply the distance law
+    (use a GainTransform for that)
+
+    [1] https://github.com/LCAV/pyroomacoustics
+    """
+
+    supports_multichannel = True
+
+    # Table of air absorption coefficients adapted from `pyroomacoustics`.
+    # The last two columns have been extrapolated with `curve_fit`
+    air_absorption_table = {
+        "10C_30-50%": [
+            x * 1e-3 for x in [0.1, 0.2, 0.5, 1.1, 2.7, 9.4, 29.0, 91.5, 289.0]
+        ],
+        "10C_50-70%": [
+            x * 1e-3 for x in [0.1, 0.2, 0.5, 0.8, 1.8, 5.9, 21.1, 76.6, 280.2]
+        ],
+        "10C_70-90%": [
+            x * 1e-3 for x in [0.1, 0.2, 0.5, 0.7, 1.4, 4.4, 15.8, 58.0, 214.9]
+        ],
+        "20C_30-50%": [
+            x * 1e-3 for x in [0.1, 0.3, 0.6, 1.0, 1.9, 5.8, 20.3, 72.3, 259.9]
+        ],
+        "20C_50-70%": [
+            x * 1e-3 for x in [0.1, 0.3, 0.6, 1.0, 1.7, 4.1, 13.5, 44.4, 148.7]
+        ],
+        "20C_70-90%": [
+            x * 1e-3 for x in [0.1, 0.3, 0.6, 1.1, 1.7, 3.5, 10.6, 31.2, 93.8]
+        ],
+        "center_freqs": [125, 250, 500, 1000, 2000, 4000, 8000, 16000, 32000],
+    }
+
+    def __init__(
+        self,
+        min_temperature=10.0,
+        max_temperature=10.0,
+        min_humidity=30,
+        max_humidity=90,
+        min_distance=10,
+        max_distance=100,
+        n_fft=256,
```

Comment:
There's a big chance the user won't take the time to understand this and set it correctly. Maybe it can be decided automatically (by a specific fraction of the sample rate, then ceil to nearest power of two) instead of being a parameter?

Reply:
I changed it so that its computed automatically so that the first octave band (0-250Hz) can be shown as a single fft bin.